packages/disploy/src/structs/GuildTextChannel.ts

```diff
@@ -18,7 +18,7 @@ export class GuildTextChannel extends BaseChannel {
 	 */
 	public type: ChannelType.GuildText = ChannelType.GuildText;
 
-	public constructor(app: App, raw: APIGuildChannel<ChannelType.GuildText>) {
+	public constructor(app: App, public override raw: APIGuildChannel<ChannelType.GuildText>) {
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.